core/platform/bulk_email/mailchimp_bulk_email_services.py

```diff
@@ -76,6 +76,40 @@ def _get_mailchimp_class():
         mc_api=feconf.MAILCHIMP_API_KEY, mc_user=feconf.MAILCHIMP_USERNAME)
 
 
+def _create_user_in_mailchimp_db(user_email):
+    """Creates a new user in the mailchimp database and handles the case where
+    the user was permanently deleted from the database.
+
+    Args:
+        user_email: str. Email ID of the user. Email is used to uniquely
+            identify the user in the mailchimp DB.
+
+    Raises:
+        Exception. Any error (other than the one mentioned below) raised by the
+            mailchimp API.
+
+    Returns:
+        bool. False if the user was permanently deleted earlier and therefore
```

Comment:
`bool. Whether the user was successfully added to the db. (This will be False if the user was permanently deleted earlier and therefore cannot be added back.)`

Reply:
Done